README.md

```diff
@@ -12,6 +14,113 @@ Add the following line to your `deps` in `mix.exs`. Run `mix deps.get`.
 {:membrane_element_udp, "~> 0.2"}
 ```
 
+## Usage example
+
+The example below shows 2 pipelines - `UDPDemo.Send` downloads an example file over HTTP and
+sends it over UDP socket via localhost:
+
+```elixir
+defmodule UDPDemo.Send do
+  use Membrane.Pipeline
+
+  alias Membrane.Element.{Hackney, UDP}
+  alias Membrane.Pipeline.Spec
+
+  def handle_init(_) do
+    children = [
+      source: %Hackney.Source{
+        location: "https://membraneframework.github.io/static/video-samples/test-video.h264"
+      },
+      udp: %UDP.Sink{
+        destination_address: {127, 0, 0, 1},
+        destination_port_no: 5001,
+        local_address: {127, 0, 0, 1}
+      }
+    ]
+
+    links = %{
+      {:source, :output} => {:udp, :input}
+    }
+
+    spec = %Spec{children: children, links: links}
+    {{:ok, spec: spec}, %{}}
+  end
+end
+```
+
+The `UDPDemo.Receive` retrieves packets from UDP socket and
+saves the data to the `/tmp/udp-recv.h264` file.
+
+Bear in mind that for other files/sending pipelines you may need do adjust
+[`recv_buffer_size`](https://hexdocs.pm/membrane_element_udp/Membrane.Element.UDP.Source.html#module-element-options)
+option  in `Membrane.Element.UDP.Source` that determines the maximum size of received packets.
+
+```elixir
+defmodule UDPDemo.Receive do
+  use Membrane.Pipeline
+
+  alias Membrane.Element.{File, UDP}
+  alias Membrane.Pipeline.Spec
+
+  def handle_init(_) do
+    children = [
+      udp: %UDP.Source{
+        local_address: {127, 0, 0, 1},
+        local_port_no: 5001
+      },
+      sink: %File.Sink{
+        location: "/tmp/udp-recv.h264"
+      }
+    ]
+
+    links = %{
+      {:udp, :output} => {:sink, :input}
+    }
+
+    spec = %Spec{children: children, links: links}
+    {{:ok, spec: spec}, %{}}
+  end
+end
+```
+
+The snippet below presents how to run these pipelines. For convenience it uses `Membrane.Testing.Pipeline`
+that wraps the pipeline modules above and allows to assert on state changes and end of stream events from the elements.
+
+```elixir
+alias Membrane.Testing.Pipeline
+import Membrane.Testing.Assertions
+
+{:ok, sender} = Pipeline.start_link(%Pipeline.Options{module: UDPDemo.Send})
+{:ok, recv} = Pipeline.start_link(%Pipeline.Options{module: UDPDemo.Receive})
```

Comment:
I would follow your convention and call this process variable `receiver` but that is a nitpick.